docs/guides/StandarizedUnitsNaming.md

```diff
@@ -0,0 +1,115 @@
+# Standardowe nazewnictwo jednostek
+
+
+
+## Wstęp + rozpiska komunikacji radiowej
+
+W Armaforces przyjęło się stosowanie zwyczajowego, standardowego nazewnictwa jednostek. Należy tu zaznaczyć, że zwyczajowe nazewnictwo nie jest obowiązkowe (nie jest wymuszane na missionmakerach). Missionmaker może wprowadzić inne nazewnictwo w swojej misji. 
+
+Przykładowo, w przeszłosci występowały różnice w nazewnictwie, np.:
+- jednostki były nazywane zupełnie inaczej, np. w celu zachowania klimatu misji (np. Alpha -> Anton, Bravo -> Boris, itd.),
+- rzadziej używane jednostki (inne niż np. Alpha, Bravo, Kilo, November, Zulu, itd.) miały inne funkcje niż zazwyczaj (np. Mike był transportem lądowym zamiast powietrznym).
+
+W związku z powyższym, należy przyjmować poniższe nazewnictwo wyłącznie jako "dobre praktyki". 
+
+Adekwatnie, standardowy sposób komunikacji na misji, używając radia z ACRE2, dostępny jest tutaj: 
+[Standardowa rozpiska komunikacji](https://docs.google.com/spreadsheets/d/1gDA2t6YibfjD4XZMrUL5eZU3Ge1cIQzBiBfBN6-AW08/edit#gid=1772918807)
+
+<u>Stosowane skróty</u>:
+
+**ATC** - Air Traffic Control
+
+**BPP** - Biedna Pie*dolona Piechota
+
+**CAS** - Close Air Support
+
+**FAC** - <span style="color:red">Forward Air Controller</span> <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+
+**JFO** - <span style="color:red">Joint Fire Observer</span> <!-- prośba o potwierdzenie, potencjalnie wymagana dyskusja -->
+
+**JTAC** - Joint Terminal Attack Controller
+
+**MEDEVAC** - MEDical EVACuation
+
+**NCO** - Non-Commissioned Officer
```

Comment:
Don't use HTML in Markdown, it makes raw document look like crap and is completely unnecessary.

Change this section to be a proper unordered list in Markdown rather than having each element in a separate paragraph without any real reason.

I'd be for removing BPP from this page at all, it's a bit too internal term to be used in official documentation. And stick to abbreviations we do use rather than listing all common ones even if we don't use them. And explain what they mean for us.
```suggestion
Często używane skróty:

- CAS - _Close Air Support_, wsparcie z powietrza w śmigłowcu lub samolocie
- JTAC - _Joint Terminal Attack Controller_, osoba koordynująca działania jednostek powietrznych podczas operacji
- MEDEVAC - _MEDical EVACuation_, specjalna jednostka zapewniająca wsparcie medyczne oraz szybką ewakuację rannych, najczęściej śmigłowcem
```

Reply:
Removed HTML references.